script/Ciel/Day 9/09_01_CIEL09_9.txt

```diff
@@ -557,27 +569,28 @@ Eternity was already there.
 [sha:950694fe8f8e1df0b62c9a704a5432ca66476347]{
 -- Page 796, Offset 24250.
 -- 否。永遠に留めておきたいと思ってしまったモノが、そこにあった。
-Nay; something I wanted to retain in eternity was there.
+Nay; something I wanted to %{i}retain%{/i} for eternity was there.
 }
 [sha:97caf271dfa47cc7c7f05479de93be7282733280]{
 -- Page 797, Offset 24251.
 -- 今でも―――何度この世から退場しようと、
-Even now――no matter how many times I may depart this world
+// Sentences shuffled to avoid break
+It doesn't matter how many times I depart this world.
 }
 [sha:ca750cf7e755ee7481920972c97d1c2afea8befa]{
 -- Page 797, Offset 24252.
 -- 身を切り裂くほどの憎悪と歓びが繰り返される。
-――does hatred and joy enough to tear my body into pieces recur within me.
+The hatred and joy I feel each time is still strong enough to tear my body to shreds.
 }
 [sha:bd89590c28da63ffcc97a69588f9c1484fd23409]{
 -- Page 797, Offset 24253.
 -- あの時。世界の全てを知る事と、あの女を知る事を、私は<秤|はかり>にかけてしまった。
-Back then, I put 'knowing all of the world' and 'knowing her' against each other on a scale.
+Back then, I placed 'knowing all of the world' and 'knowing her' on each side of a set of scales.
```

Comment:
"against each other on a scale" is the collocation, as far as I understand.

Reply:
Maybe if you're ranking things on a scale (abstract), but I think this should either be a literal _set_ of scales as written, or just fully idiomized to
> Back then, I weighed 'knowing all of the world' against 'knowing her'.

Reply:
I do prefer that, though I genuinely think (and I see a bunch of example of this on google) that if you say you weigh two things against each other on a scale, people will only ever get the oldey timey scale imagery in their head.